api3/common/openapi.yaml

```diff
@@ -395,6 +395,12 @@ components:
       type: integer
       format: int64
   parameters:
+    ApiID:
+      name: apiID
+      description: "The path parameter for apiID type"
+      in: path
+      schema:
+        $ref: '#/components/schemas/ApiID'
```

Comment:
We had this before, but for some reason the code generation was failing. I don't know if it's the case, but we decided not to have the ApiID defined as a common parameter section.

Reply:
fixed